gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/SessionTimeout.java

```diff
@@ -15,26 +15,31 @@ public class SessionTimeout implements Serializable {
 	private int refreshLimit = 0;
 	private String method = "";
 	private String additionalData = "";
+	private String signInUrl = "";

```

Comment:
Changelog should be updated (I don't remember if we used to do this as we go, but probably a good idea to do so)

Reply:
done